agents/VapixAgent.ts

```diff
@@ -0,0 +1,262 @@
+import { Logger } from 'dap-utils/Logger/Logger';
+import { XMLParser } from 'fast-xml-parser';
+
+import { TSDCardInfo } from 'AgentFactoryProvider/types/AcapReact';
+import { TCameraPTZItem, TCameraPTZItemData, TPtzOverview } from 'AgentFactoryProvider/types/CSw/Camera';
+import { TCameraSupportedAudioSampleRates } from 'AgentFactoryProvider/types/CSw/MasterCamera';
+
+import { arrayToUrl, parseCameraPTZfromReq } from './Clients/utils';
+
+import type { ProxyClient, TProxyParam } from './Client';
+export class VapixAgent {
+    protected _baseAddress = '/axis-cgi';
+
+    constructor(private client: ProxyClient, private logger: Logger) {}
+
+    async getParametr(paramNames: string | string[], proxy: TProxyParam = null): Promise<Record<string, string>> {
```

Comment:
*getParameter - osobne bych sem ty agenty zatim nepridaval, neni to dotazene a podobne funkce ted resi trida CameraVapix